dockers/docker-platform-monitor/Dockerfile

```diff
@@ -18,4 +18,5 @@ ENTRYPOINT /usr/bin/config.sh       \
     && service lm-sensors start     \
     && service smartmontools start  \
     && service sensord start        \
+    && service fancontrol restart   \
```

Comment:
this fan control logic is probably difficult to be unified across different platform. other platform can have their own fan control mechanism/program. 

our current thinking is to leave this decision to the platform owner to choose the best mechanism to control the fan speed.  therefore, it is better not to put this into platform-montior docker since different platform owners may choose different mechanism. therefore, we'd recommend to move this into your platform debian package, which will be installed based on the hardware platform.